boblogistics/prototypes/entity/pipes.lua

```diff
@@ -13,6 +20,16 @@ function bobmods.logistics.set_pipe_distance(pipe, level)
   end
 end
 
+function bobmods.logistics.set_pipe_height(pipe, level)
+  if (data.raw["pipe"][pipe]) then
+    data.raw["pipe"][pipe].fluid_box.height = bobmods.logistics.pipe_height(level)
+  end
+  if (data.raw["pipe-to-ground"][pipe]) then
```

Comment:
```suggestion
  if data.raw["pipe-to-ground"][pipe] then
```
same as previous comment, matching the file's existing styles